modules/administration_manual/pages/configuration/files/file_sharing_configuration.adoc

```diff
@@ -2,8 +2,8 @@
 
 The sharing policy is configured on the Admin page in the *"Sharing"* section.
 
-NOTE: If you don't see the sharing section, try disabling your AdBlock browser plugin. 
-      It might also be related to another installed ad blocker in your browser. 
+NOTE: If you don't see the sharing section, try disabling your AdBlock browser plugin.
```

Comment:
`NOTE:` --> `[NOTE:]`
(to render properly)

Reply:
Same here... And I have not checked how this change will render compared to now.

Reply:
As above for TIP.